macaw/interface/interface.py

```diff
@@ -16,5 +18,5 @@ def run(self):
         pass
 
     @abstractmethod
-    def result_presentation(self, response_msg, params):
+    def result_presentation(self, response_msg: Message, params: dict):
```

Comment:
`Dict`

Reply:
It's equal if I am not specifying key and value types explicitly.